Add support to metadata that is automatically added by VCD (#1146)
vmware/terraform-provider-vcd: .changes/v3.11.0/1146-improvements.md

* Add support to the metadata that gets automatically created on `vcd_vapp_vm` and `vcd_vm` when they are created by a VM from a vApp Template,
  with the new `inherited_metadata` block. Example of metadata entries of this kind: `vm.origin.id`, `vm.origin.name`, `vm.origin.type` [GH-1146]

> * Add support to the metadata that gets automatically created on `vcd_vapp` when they are created by a vApp Template,

Reviewer: ```suggestion
* Add support to the metadata that gets automatically created on `vcd_vapp` when it is created by a vApp Template or another vApp,
```

Author: Fixed, thanks!
